### DISCUSSION TITLE
Ignore playlist files beginning with "."

### DISCUSSION THREAD

**Discussion by crosbyh** [navidrome/navidrome#2366] [Ideas]
I opened some of my .m3u playlists in macOS, which creates metadata files beginning with "._" for every file opened from a network share, which contains some extended file system attributes. Not a desirable behavior of macOS when working across platforms, and my understanding is that it's difficult to work around within macOS.

These ._ files now show up as duplicate playlists in Navidrome when it scans the folder containing the playlists.

I am wondering if it is possible to add a setting for Navidrome to **ignore invisible playlist files** when it scans the music directory. 

Thank you!

**Comment by deluan** [comment]
Good catch! It already skip songs that start with this prefix, but I forgot to extend this to playlists as well. Will convert this to an issue.

**Reply by crosbyh** [reply]
Thank you!

**Comment by deluan** [comment]
Fixed in #2390
